src/baldr/graphreader.cc

```diff
@@ -851,5 +851,10 @@ AABB2<PointLL> GraphReader::GetMinimumBoundingBox(const AABB2<PointLL>& bb) {
   return min_bb;
 }
 
+int GraphReader::GetTimezone(const baldr::GraphId& node, const GraphTile*& tile) {
```

Comment:
this was a helper funciton in the timedependent algorithms but since all of them need to use it i figured we can move it to the graphreader where we have other helpers